app/domain/authentication/usecase/create_token.go

```diff
@@ -0,0 +1,51 @@
+package usecase
+
+import (
+	"github.com/golang-jwt/jwt"
+	"github.com/patriciapedrosaa/transfer-me/app/domain/entities"
+	"os"
+)
+
+func (a Authentication) CreateToken(login LoginInputs) (string, error) {
+	validUser, err := a.CheckLogin(login)
+	if !validUser {
+		return "", err
+	}
+
+	account, err := a.accountRepository.GetByCpf(login.CPF)
+	if err != nil {
+		return "", err
+	}
+
+	token, err := entities.NewCreateToken(account.Name, account.AccountID)
+	if err != nil {
+		return "", err
+	}
+
+	err = a.authenticationRepository.CreateToken(token)
+	if err != nil {
+		return "", err
+	}
+
+	err = os.Setenv("ACCESS_SECRET", "dons deed crop fame blat lacy")
+	if err != nil {
+		return "", err
+	}
```

Comment:
this information is used across the use cases, what do you think of taking it as an attribute of UseCase and inject it since you opted to read it from an env var?